errata.md

```diff
@@ -16,6 +16,9 @@ dated 2019-06-02:
 * Section 3.3.2 and 3.3.3 headings (page 71) - the headings go past
   the intended margin (though they're quite readable).
 * Section 3.9.3 (page 97) - "appraoch" should be "approach".
+* Section 3.10 (page 103 and 104) - the output of "show statement ax-1/full"
+  seems to be shown twice (once at the end of page 103, and once at the top of
```

Comment:
No reason to be coy. It *is* shown twice :-).

Reply:
Hehe okay. Updated to be less coy.